operator/v1/types_ingress.go

```diff
@@ -1453,6 +1453,30 @@ type IngressControllerTuningOptions struct {
 	// +kubebuilder:validation:Optional
 	// +optional
 	MaxConnections int32 `json:"maxConnections,omitempty"`
+
+	// reloadInterval defines the minimum interval at which the router is allowed to reload
+	// to accept new changes. Increasing this value can prevent the accumulation of
```

Comment:
do these changes include new routes?  If this value were set to 30 days and a new route was created, would it be served?  If not, is there a way in the API to know that this has happened?

Reply:
From Miciah: "`RELOAD_INTERVAL` does affect new routes.  There is no direct feedback to the user on the route status or otherwise whether the HAProxy configuration has been updated and reloaded."

Reply:
Following up on this, is the current solution acceptable?

Reply:
> Following up on this, is the current solution acceptable?

Given a max of 120s, yes.  If it ever gets extended, the route API needs to grow a way to represent that.